src/clarity-angular/datagrid/datagrid.ts

```diff
@@ -116,6 +116,10 @@ export class Datagrid implements AfterContentInit, AfterViewInit, OnDestroy {
         this.refresh.emit(state);
     }
 
+    public hasFilters(): boolean {
+        return this.filters.getActiveFilters().length > 0 ? true : false;
```

Comment:
I dont think so @mathisscott. I will confirm with @youdz once. I think the filters might be empty but not null.